crypto/bn254/keystore/keystore.go

```diff
@@ -0,0 +1,32 @@
+package keystore
+
+import (
+	"context"
+	"github.com/Layr-Labs/eigensdk-go/crypto/bls"
+	"github.com/Layr-Labs/eigensdk-go/types"
+)
+
+type Signer struct {
+	keyPair *bls.KeyPair
+}
+
+func New(path, password string) (*Signer, error) {
+	keyPair, err := bls.ReadPrivateKeyFromFile(path, password)
```

Comment:
this will need to be updated to handle the new format from https://github.com/Layr-Labs/bn254-keystore-go - will add that.